### DISCUSSION TITLE
[委托]物流事故!急寻绳匠!

### DISCUSSION THREAD

**Discussion by waerte-yang** [share121/inter-knot#3075] [其他]
![IMG_20240819_162228](https://github.com/user-attachments/assets/23c69be3-1e95-46fe-8b9f-5974d79fb672)
我订购的食材在运输途中出了状况，「天马速运」的订单显示，货车一头闯进了十四分街的伴生空洞里，到现在也没出来。
请绳匠帮我去看看货车的情况，食材拿不拿得回来已经不重要了，只希望货车司机没逝！

**Comment by homteam** [comment]
这个委托我接了！

**Comment by cleo66** [comment]
这个委托我不能接！

**Comment by fairy-fire** [comment]
这个委托我接了

**Comment by yzysrewrwqr** [comment]
这个委托我接不接？

**Comment by KUFCYe** [comment]
这个委托我接了

**Comment by yivitte** [comment]
这个委托我接了

**Comment by ALXY-12** [comment]
这个委托我接了！

**Comment by XuemeiLing** [comment]
这个委托我接了！

**Comment by baile1** [comment]
这个委托我接了！

**Comment by chaojituentuenshu** [comment]
这个委托我接了！

**Comment by jxjw** [comment]
这个委托我接了！

**Comment by Aha-SWYH** [comment]
这个委托被人接了！！！我（新艾利都优美语言）

**Comment by fallingnight** [comment]
这个委托，啊嘞，已经被人接了吗0.0

**Comment by NT-J** [comment]
这个委托我记得接过了！

**Comment by qqqqqq77777** [comment]
这个委托我不接！<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >

**Comment by alannnnnn22** [comment]
天马就是不行，已经是第二次了

**Comment by haxyyx** [comment]
总感觉你们公司物流不是很安全，接二连三多少次事故了！
